lambda_functions/rds_init/lambda-handler.py

```diff
@@ -0,0 +1,51 @@
+import json
+import logging
+import os
+import sys
+
+import psycopg2
+from aws_lambda_powertools.utilities import parameters
+from aws_lambda_powertools.utilities.typing import LambdaContext
+
+# This lambda function is only meant to be run once during cdk initialization,
+# as post-db creation initialization script
+
+bde_host = os.environ["BDE_HOST"]
+bde_ro_user = os.environ["BDE_RO_USER"]
+# bde_ro_cred = parameters.get_secret(os.environ["BDE_RO_USER_CRED"])
+
+rds_fdw_host = os.environ["RDS_FDW_HOST"]
+rds_fdw_db = os.environ["RDS_FDW_DB"]
+
+rds_fdw_root_cred = parameters.get_secret(os.environ["RDS_FDW_ROOT"])
```

Comment:
Should this be using secrets manager?

Reply:
Yes. The root credentials for the RDS with FDW should be in the secrets manager. Am I missing / misunderstood something?

Reply:
Is that what `parameters` does? If so, I've misunderstood. This looked like it was getting from somewhere else.